model-optimizer/automation/package_BOM.txt

```diff
@@ -469,6 +472,7 @@ extensions/front/tf/rfcn_support_api_v1.10.json
 extensions/front/tf/rfcn_support_api_v1.13.json
 extensions/front/tf/rfcn_support_api_v1.14.json
 extensions/front/tf/roll_ext.py
+extensions/front/tf/RollRealImagPackReplacement.py
```

Comment:
Please, remove word "Replacement" from the file names and from class names.

Reply:
Done.